phpBB/phpbb/notification/type/forum.php

```diff
@@ -136,9 +136,9 @@ public function get_email_template_variables()
 			'TOPIC_TITLE'				=> htmlspecialchars_decode(censor_text($this->get_data('topic_title')), ENT_COMPAT),
 
 			'U_VIEW_POST'				=> generate_board_url() . "/viewtopic.{$this->php_ext}?p={$this->item_id}#p{$this->item_id}",
-			'U_NEWEST_POST'				=> generate_board_url() . "/viewtopic.{$this->php_ext}?f={$this->get_data('forum_id')}&t={$this->item_parent_id}&e=1&view=unread#unread",
-			'U_TOPIC'					=> generate_board_url() . "/viewtopic.{$this->php_ext}?f={$this->get_data('forum_id')}&t={$this->item_parent_id}",
-			'U_VIEW_TOPIC'				=> generate_board_url() . "/viewtopic.{$this->php_ext}?f={$this->get_data('forum_id')}&t={$this->item_parent_id}",
+			'U_NEWEST_POST'				=> generate_board_url() . "/viewtopic.{$this->php_ext}?t={$this->item_parent_id}&e=1&view=unread#unread",
+			'U_TOPIC'					=> generate_board_url() . "/viewtopic.{$this->php_ext}?t={$this->item_parent_id}",
+			'U_VIEW_TOPIC'				=> generate_board_url() . "/viewtopic.{$this->php_ext}?t={$this->item_parent_id}",
 			'U_FORUM'					=> generate_board_url() . "/viewforum.{$this->php_ext}?f={$this->get_data('forum_id')}",
 			'U_STOP_WATCHING_FORUM'		=> generate_board_url() . "/viewtopic.{$this->php_ext}?uid={$this->user_id}&f={$this->get_data('forum_id')}&t={$this->item_parent_id}&unwatch=forum",
```

Comment:
I'm not sure in this case, it is to unwatch the forum but uses `viewtopic.php` 🤔. The unwatch link for new forum topics looks good:
```
If you no longer wish to watch this forum click the following link:
https://example.org/phpbb/viewforum.php?uid=123&f=12&unwatch=forum
```

The above is however used with "new reply in forum" notifications:
```
If you no longer wish to watch this forum click the following link:
https://example.org/phpbb/viewtopic.php?uid=123&f=12&t=1234&unwatch=forum
```

Opposed to the dedicated "new reply in topic" notification, where it is really to unwatch the topic:
```
If you no longer wish to watch this topic click the following link:
https://example.org/phpbb/viewtopic.php?uid=123&t=1234&unwatch=topic
```

So to match text and since it is indeed about a forum reply, not a topic reply, I think it should be:
```suggestion
			'U_STOP_WATCHING_FORUM'		=> generate_board_url() . "/viewforum.{$this->php_ext}?uid={$this->user_id}&f={$this->get_data('forum_id')}&unwatch=forum",
```
isn't it?